client/datasync-client/package.json

```diff
@@ -22,6 +22,11 @@
     "require": [
       "ts-node/register"
     ],
+    "reporter": [
+      "lcov",
+      "text"
```

Comment:
We should really have both. Lcov is required for the generated reporting, and text is require for terminal output. I'll add in the  missing `text` reporter.